tests/sanic/test_view.py

```diff
@@ -0,0 +1,117 @@
+from random import random
+
+import pytest
+
+import strawberry
+from sanic import Sanic
+from strawberry.sanic.views import GraphQLView as BaseGraphQLView
+from strawberry.types import ExecutionResult, Info
```

Comment:
Was isort run here? If so, we may need to inspect our configuration, as the strawberry imports are supposed to be separated from the 3rd party ones.

Reply:
This applies to `strawberry/sanic/views.py` as well

Reply:
Yes, pre-commit was run and it actually moves the `sanic` import after `import strawberry`.

Reply:
this is so strange :D I'll see if we can improve it, for now let's leave it as it is :)